src/packet/pacer.rs

```diff
@@ -431,25 +431,32 @@ impl LeakyBucketPacer {
             .filter_map(|qs| (qs.snapshot.first_unsent.map(|t| (t, qs))))
             .min_by_key(|(t, _)| *t);
 
-        // Unpaced packets (such as audio by default) are sent immediately.
+        // Unpaced packets (such as audio buy default) are sent immediately.
```

Comment:
typo?

Reply:
Well that is strange.   I didn't even try and change that line.  Oh well.  I reverted that line.